src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -555,6 +576,23 @@ const PlrDaoStakingTransactionBlock = ({
     receiverAddress,
   ]);
 
+  const setValueForUnStaking = async () => {
```

Comment:
Generally in react set as a prefix is used for state variable setting.
We may change the name to something else

Reply:
Yeah agreed, also unstaking should be one word